packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -164,7 +164,7 @@ const APPLICATION_EXTENSION_QUESTIONS = [
         name: 'project.extensionName',
         message:
             'What is the name of your Application Extension? \n' +
-            'The name must follow the pattern "@{namespace}/extension-{package-name}", where namespace is optional.',
+            'The name must follow the pattern "@{namespace}/{package-name}", where namespace is optional.',
```

Comment:
I wonder if we should simplify this. In our pattern we don't mention that you can't use some special characters. But if we use the package above we can probably say something like "The name must follow standard npm package name conventions"